benchmark/benchmark.py

```diff
@@ -0,0 +1,224 @@
+# flake8: noqa: E402
+import argparse
+import bz2
+import csv
+import datetime
+import os
+import re
+import sys
+from ast import literal_eval as to_list
+from io import StringIO
+from pathlib import Path
+
+import pandas as pd
+from matplotlib import pyplot as plt
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from eyecite import get_citations
+
+csv.field_size_limit(sys.maxsize)
+
+root = Path(__file__).parent.absolute()
+
+MAX_ROWS_IN_MD = 51
+
+
+class Benchmark(object):
+    """Benchmark the different eyecite branches"""
+
+    def __init__(self):
+        self.now = datetime.datetime.now()
+        self.times = []
+        self.totals = []
+        self.list_of_ids = []
+        self.opinions = []
+        self.count = 0
+        self.fields = []
+        self.gains = []
+        self.losses = []
+
+    def get_filepath(self, filename):
+        return Path.joinpath(root, filename)
+
+    def fetch_citations(self, row) -> None:
+        """Fetch citations from rows opinion data
+
+        return: None
+        """
+        row_id = row[0]
+        row = dict(zip(self.fields, row))
+        # Find opinions to test against from the zip file and return
+        # citations found
+        non_empty_rows = [
+            row[field] for field in self.fields if type(row[field]) == str
+        ]
```

Comment:
Not sure what's happening here. A comment, perhaps?

Reply:
noted.